handlers_global.go

```diff
@@ -10,26 +11,26 @@ import (
 	"time"
 
 	"github.com/stripe/veneur/samplers"
-
-	"golang.org/x/net/context"
+	"github.com/stripe/veneur/ssf"
+	"github.com/stripe/veneur/trace"
 )
 
 type contextHandler func(c context.Context, w http.ResponseWriter, r *http.Request)
 
 func (ch contextHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	// nil context is bad form, but since we don't use it, it's a quick hack
-	// to allow us to write tests before we upgrade to Go 1.7 Context type
-	// and the newer goji
-	// TODO(aditya) actually update this
-	ch(nil, w, r)
+	ctx := context.Background()
```

Comment:
i know this is premature, but it'd be _super cool_ if the local instance reported a trace id in its import request, and then the global instance recovered the trace id and could attach its trace of running the import to the trace of the local instance performing the flush. maybe add as TODO?